VAMobile/src/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -61,7 +60,7 @@ import SandboxScreen from './ProfileScreen/SettingsScreen/DeveloperScreen/Sandbo
 import ManageYourAccount from './ProfileScreen/SettingsScreen/ManageYourAccount/ManageYourAccount'
 import NotificationsSettingsScreen from './ProfileScreen/SettingsScreen/NotificationsSettingsScreen/NotificationsSettingsScreen'
 
-const { WEBVIEW_URL_CORONA_FAQ, WEBVIEW_URL_FACILITY_LOCATOR } = getEnv()
```

Comment:
do we need to also clean up this URL from the environment variables?

Reply:
The Covid 19 URL is also used on the HealthScreen, so we'll still need it